tutorial/gt4py/boilerplate.py

```diff
@@ -0,0 +1,66 @@
+from ndsl.dsl.dace.dace_config import DaceConfig, DaCeOrchestration
+from ndsl.dsl.stencil import GridIndexing, StencilConfig, StencilFactory
+from ndsl.dsl.stencil_config import CompilationConfig, RunMode
+import matplotlib.pyplot as plt
+
+
+def get_one_tile_factory(nx, ny, nz, nhalo, backend) -> StencilFactory:
```

Comment:
There's a better version of that boilerplate in the AD_example. I am also tempted by pulling those in `ndsl` itself

Reply:
Let me know where I can find the updated boilerplate.